<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgNodeQueryPathEditor</class>
 <widget class="QWidget" name="WdgNodeQueryPathEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>37</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonCurrentLayer">
     <property name="text">
      <string>current layer</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonCustomPath">
     <property name="text">
      <string>path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="klineeditPath"/>
   </item>
   <item>
    <widget class="KPushButton" name="kpushbutton"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>77</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
